kuksa/README.md

```diff
@@ -0,0 +1,124 @@
+![Kanto logo](https://github.com/eclipse-kanto/kanto/raw/main/logo/kanto.svg)
+
+# Eclipse Kanto - Eclipse Kuksa Integration
+
+# Introduction
+
+This is an example application that connects to [Eclipse Kuksa Databroker](https://github.com/eclipse-kuksa/kuksa-databroker) and demonstrates how the COVESA Vehicle Signal Specification(VSS) data could be transformed an AWS IoT Shadow using Eclipse Kanto.
```

Comment:
This should be explicitly stated that this application will function the same with different cloud provider, and this guide is specifically for AWS